src/components/header/PatientHeader.jsx

```diff
@@ -25,15 +25,15 @@ export class PatientHeader extends PureComponent {
 
   componentDidMount() {
     if (this.state.patient) {
-      const { identifiersToDisplay, identifierTypesToDisplay, currentLocationPrefix, cccNumber, hccNumber } = this.props;
+      const { identifiersToDisplay, identifierTypesToDisplay, locations, currentLocation, cccNumber, hccNumber } = this.props;
```

Comment:
we still need to remove the cccNumber and hccNumber references from react-components and just load them in APZU-IC3